sdk/order-planner-common/src/actions/BaseAction.ts

```diff
@@ -13,6 +13,7 @@ import { ActionConfig, ActionCall } from './Types'
  *              the versioned name of the action.
  */
 export abstract class BaseAction {
+  private DefaultParamsMapping: number[] = [0, 0, 0, 0]
```

Comment:
Yeah, I think so, but I don't fully understand the concern. We can talk about it.

The reason for this is that the params mapping is optional, and I need to give that array to `encodeFunctionData`, otherwise the params mapping are not included and the contract call will fail. I decided to add it here because it is just used locally. If not convinced, let's talk :)

I will mark is as readonly, though, as it was missing